test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
OK, thanks! (the "change key" system we came up with 29 years ago was long before such tools, and old habits die hard!)